importScripts/sql/transformations/LibraryExportView.sql

```diff
@@ -1,59 +1,91 @@
+DROP FUNCTION IF EXISTS clean_spaces;
+CREATE FUNCTION clean_spaces(str TEXT)
```

Comment:
There were some strings with upwards of 50 spaces, and I felt this was better than using:
`REPLACE(REPLACE(REPLACE(REPLACE(REPLACE(REPLACE(REPLACE(REPLACE(REPLACE(REPLACE(REPLACE(REPLACE(REPLACE(REPLACE(REPLACE(REPLACE(REPLACE(REPLACE(` etc.